test/regression/sql/test_all_types.sql

```diff
@@ -23,5 +23,11 @@ SELECT * exclude(
     fixed_array_of_int_list,
     list_of_fixed_int_array,
     nested_int_array, -- The nested array has different lengths, which is not possible in PG
+    date,
```

Comment:
Let's add a comment explaining why we exclude these
```suggestion
    -- the min/max values of dates/timestamps overflow in Postgres so selecting these would throw an error
    date,
```